.github/workflows/build_windows.yaml

```diff
@@ -73,7 +73,7 @@ jobs:
                       ./.hak
 
             - name: Set up build tools
-              uses: ilammy/msvc-dev-cmd@v1
+              uses: ilammy/msvc-dev-cmd@cec98b9d092141f74527d0afa6feb2af698cfe89
```

Comment:
ftr this is v1.12.1 rather than v1, which probably *is* deliberate since v1 is older

Reply:
Accidental but lets go with deliberate